src/App.jsx

```diff
@@ -0,0 +1,78 @@
+import {useState} from 'react'
+import "./App.css";
+import Contacts from './contacts.json'
+
+
+
+function App() {
+
+  const contactList = Contacts.slice(0,5)
+  const remainingContacts = Contacts.slice(5)
+  // console.log(remainingContacts)
+  // console.log(contactList)
+
+  const [contacts, setContacts] = useState(contactList)
+
+  function getRandomContact(){
+    const copyContacts = [...contacts]
```

Comment:
Here we need just a guard in order to stop adding when the remaing are 0

```suggestion
  if (remainingContacts.length === 0) {
      console.log("No more contacts can be add")
      return
    }
    const copyContacts = [...contacts]
```